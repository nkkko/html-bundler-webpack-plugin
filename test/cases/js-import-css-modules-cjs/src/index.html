<!DOCTYPE html>
<html>
<head>
  <title>CJS</title>
  <script src="./main.js" defer="defer"></script>
</head>
<body>
  <div id="root"></div>
</body>
</html>