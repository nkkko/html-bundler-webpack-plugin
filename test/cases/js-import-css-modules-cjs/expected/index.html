<!DOCTYPE html>
<html>
<head>
  <title>CJS</title>
  <link href="css/main.fa2662a2.css" rel="stylesheet">
<script src="js/main.09b19148.js" defer="defer"></script>
</head>
<body>
  <div id="root"></div>
</body>
</html>